aii-core/src/main/perl/Shellfe.pm

```diff
@@ -80,11 +83,23 @@ use constant LIVECDMETHOD    => 'Livecd';
 Readonly our $PROTECTED_COMMANDS => 'remove|configure|(re)?install';
 Readonly our $PROTECTED_OPTION   => 'confirm';
 
+# Keep this list in sync with the options list (to support no<pluginname>)
+# this is also the order in which the plugins run (in iter_plugins)
+# TODO: no dhcp? (but there's a nodhcp option)
```

Comment:
Do we need an issue for this TODO?

Reply:
I'd open an issue for this, but I'm not sure what it is really asking.

Reply:
@stdweird I'm still not sure what this TODO is about. Is it that `dhcp` is not in the list of plugin names?